content/doc/book/updating/_update-wizard.adoc

```diff
@@ -0,0 +1,66 @@
+////
+This file is only meant to be included as a snippet in other documents.
+////
+
+[[post-upgrade-wizard]]
+
+== How to safely upgrade to a new version
+
+---
+video::fEO7dOxHvgU[youtube,width=852,height=480]
+---
+
+== Post-update setup wizard
+
+There are a few post upgrade steps to perform.
+[unordered.stack]
+Verify the version update ::
+
+The Jenkins server is up and running.
+Let's now verify the version from the Jenkins dashboard:
+
+[.boxshadow]
+image:updating/check-jenkins-version.png[alt="Checking Version of Updated Jenkins Instance",width=80%]
+
+The upgraded Jenkins version can be seen in the bottom right corner of the Jenkins UI.
+
+[unordered.stack]
+Upgrading Jenkins Java Version From 8 to 11 ::
+Jenkins has transitioned from Java 8 to Java 11 as a baseline requirement.
+Java 17 is also supported as of the 2.361.1 Long Term Support release.
+Therefore, a best practice is to verify your Java version if you are upgrading from an older version.
+To upgrade Jenkins from Java 8 to Java 11, refer to the link:/doc/administration/requirements/upgrade-java-guidelines/[Upgrade Java Guideline]
+and link:/doc/administration/requirements/java/[Java requirements].
+
+---
+video::L2Uomz8RWUM[youtube,width=800,height=420]
+---
+
+[unordered.stack]
+Compatibility with the new version ::
+
+Plugins are one of the most crucial parts of the Jenkins pipeline.
+In fact, each job uses a plugin for its execution.
+Check the available updates for installed plugins:
+
+. Go to the Jenkins dashboard.
+. Select *Manage Jenkins* in the navigation pane on the left side of the Jenkins dashboard.
+. Select *Manage Plugins*.
+
+[.boxshadow]
+image:updating/jenkins-plugin-manager.png[alt="Jenkins plugin manager to check compatibility",width=80%]
+
+The above image shows that there is no need to update any plugins.
+
+After upgrading Jenkins, you may encounter data compatibility issues with the new version.
+To ensure that our data is in the correct format, we should update it to the correct format.
+To check the data version update, follow these steps:
```

Comment:
I think this should be a link to the existing page that describes the "Manage old data" choices and reasons.
```suggestion
Sometimes a Jenkins plugin changes the data that it is storing.
In other cases, a plugin that has been uninstalled may have left data in the system that Jenkins no longer needs.
The link:/doc/book/managing/plugins/#removing-old-data["Manage old data"] page allows administrators to remove those unused items.
```